### PR TITLE
a few updates for docs

### PR DESCRIPTION
**Pull request by wenbingl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,17 +6,14 @@
 
 
 # Introduction 
-keras2onnx enables you convert the keras models into [ONNX](https://onnx.ai).
-Initially Keras converter was developer in the project onnxmltools. To support more kinds of keras models and reduce the complexity of mixing multiple converters, keras2onnx was created to convert the keras model only.
+This Keras model converter(keras2onnx) enables you convert the keras models into [ONNX](https://onnx.ai) model format .
```
Comment (vinitra) [inline README.md]: Minor spacing and word choice suggestions for readability -- proposal sentence below:

> The keras2onnx model converter enables users to convert Keras models into the [ONNX](https://onnx.ai) model format.

File: README.md [hunk 2]
```diff
@@ -6,17 +6,14 @@
...
+Initially Keras converter was developed in the project onnxmltools. To support more kinds of keras models and reduce the complexity of mixing multiple converters, keras2onnx was created to convert the keras model only.
```
Comment (vinitra) [inline README.md]: Proposal: minor grammar fix, onnxmltools link, consistent capitalization
> Initially, the Keras converter was developed in the project [onnxmltools](http://github.com/onnx/onnxmltools).
> To support more kinds of Keras models and reduce the complexity of mixing multiple converters, keras2onnx was created to convert models from only the Keras framework.

File: README.md [hunk 3]
```diff
@@ -6,17 +6,14 @@
...
 
-keras2onnx supports the keras lambda/custom layer by parsing the TF graph built from Keras model.
+keras2onnx converter supports not only all keras built-in layers, but also the lambda/custom layer by working with [tf2onnx](https://github.com/onnx/tensorflow-onnx) converter which was embedded into the source tree directly to avoid the version conflict and install complexity.
```
Comment (vinitra) [inline README.md]: ... which _is_ embedded into ...
... to avoid _version conflicts and installation_ complexity.

**Comment by wenbingl** [comment]
@vinitra Thanks for your comments, I made the change accordingly. Feel free to update these docs if you see there is any issue.

**Pull request merged**